mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/clients/pdl/PdlOpplysning.kt

```diff
@@ -53,3 +53,30 @@ enum class PdlGradering {
     STRENGT_FORTROLIG_UTLAND,
     UGRADERT,
 }
+
+sealed class GeografiskTilknytningResponse {
+    data class GtKommune(val value: String) : GeografiskTilknytningResponse()
+    data class GtBydel(val value: String) : GeografiskTilknytningResponse()
+    data class GtUtland(val value: String?) : GeografiskTilknytningResponse()
+    data object GtUdefinert : GeografiskTilknytningResponse()
+}
+
+fun PdlGeografiskTilknytning?.toGeografiskTilknytningResponse(): GeografiskTilknytningResponse {
```

Comment:
Synes også denne ikke trengs å være en extension på `PdlGeografiskTilknytning?`, men heller `fun PdlGeografiskTilknytning.toGeografiskTilknytningResponse()`. Da blir forskjellen at vi må gjøre `it.geografiskTilknytning?.toGeografiskTilknytningResponse()`, men det blir tydligere i koden at dette feltet er nullable i modellen.